.github/CODEOWNERS

```
@@ -0,0 +1,41 @@
+# These owners will be the default owners for everything in
+# the repo. Unless a later match takes precedence,
+# @global-owner1 and @global-owner2 will be requested for
+# review when someone opens a pull request.
+*       @ZcashFoundation/network-owners @ZcashFoundation/cryptographic-owners
+
+# Network and Async Code
+# To be reviewed by Teor, Janito, or Alfredo.
+/tower-batch/                           @ZcashFoundation/network-owners
+/tower-fallback/                        @ZcashFoundation/network-owners
+/zebra-consensus/                       @ZcashFoundation/network-owners
+/zebra-network/                         @ZcashFoundation/network-owners
+/zebra-state/                           @ZcashFoundation/network-owners
```

We recently split out this crate:
```suggestion
/zebra-node-services/                  @ZcashFoundation/network-owners
/zebra-state/                           @ZcashFoundation/network-owners
```